integr8ly/tutorial-web-app-operator
# Intly 1993

adds persistent volume claim, volume and mount to the webapp template. This is to store the database for the user defined walkthrough repos.

Verification steps:

1. Deploy the webapp to a cluster using this operator version
1. Check the webapp container: it should have a persistent volume mounted named `user-walkthroughs`
1. Check the webapp env vars: there should be an env var named `DATABASE_LOCATION` pointing to this new volume.
